packages/server/.env

```diff
@@ -21,5 +21,6 @@ EMAIL_USER=rwrrrmiyvsgrhocw@ethereal.email
 EMAIL_PASSWORD=hMnwJM2Zuwhr6ZrpHe
 
 # Stripe
+STRIPE_PUBLIC_KEY=
```

Comment:
STRIPE_PUBLIC_KEY is needed only on front-end side, why do you add it here? These settings are read by the backend, not frontend